index.html

```diff
@@ -195,5 +196,6 @@
             {indicator: true}
         ));
         openmct.start();
+        window.addEventListener('beforeunload', openmct.destroy.bind(openmct));
```

Comment:
Let's move this out of index.html. Maybe into [MCT.js itself?](https://github.com/nasa/openmct/blob/master/src/MCT.js)

Reply:
Done.